docs/DeveloperGuide.md

```diff
@@ -567,64 +567,212 @@ testers are expected to do more *exploratory* testing.
+
+`settlePerson 1`
+
+Expected Output in the Transaction List Panel: A new transaction is created that is named "Settle balance with Alex".
+
+Expected Output in the Person List Panel: The balance of Alex is now "0.00"
+
+Expected Output in the Command Output Panel: Balance settled message is shown along with the name of Alex.
+
+### Sorting people by their balances
+
+Prerequisite: There are a few people stored in Spend N Split with varying balances.
+
+`sortPerson +`
+
+Expected Output in the Person List Panel: The people displayed are sorted in descending order of balance.
+
+Expected Output in the Command Output Panel: A message stating that the Person list is sorted in descending order of 
+balance is shown.
+
+`sortPerson -`
+
+Expected Output in the Person List Panel: The people displayed are sorted in ascending order of balance.
+
+Expected Output in the Command Output Panel: A message stating that the Person list is sorted in ascending order of
+balance is shown.
+
+### Adding a transaction
+
+Prerequisite: There is only 1 person stored in Spend N Split. Their name is "Alex Yeoh".
+
+`addTransaction d=bread n=Alex Yeoh c=25.00 n=Self w=1 n=Alex Yeoh w=1`
+
+Expected Output in the Transaction List Panel: New transaction is added into the list.
+
+Expected Output in the Command Output Panel: New transaction added message is shown along with the transaction's details.
+
+
+### Deleting a Transaction
+
+Prerequisite: There is at least 1 transaction in the Transaction List Panel.
+
+`deleteTransaction 1`
+
+Expected Output in the Transaction List Panel: The first transaction in the list is deleted.
+
+Expected Output in the Command Output Panel: Delete transaction message is shown along with the transaction's details.
+
+`deleteTransaction 0`
+
+Expected Output in the Command Output Panel: Error message stating that there is an invalid command format.
+### Editing a Transaction's details
+
+Prerequisite: There is at least 1 transaction in the Transaction List Panel.
+
+`editTransaction 1 c=20`
+
+Expected Output in the Transaction List Panel: The first transaction in the list has their cost changed to 20.00, and retains
+the rest of their details.
+
+Expected Output in the Command Output Panel: Edit transaction message is shown along with the transaction's details.
+
+`editTransaction 0`
+
+Expected Output in the Command Output Panel: Error message stating that there is an invalid command format.
+
+### Listing multiple transactions with matching payers and payees.
+
+Prerequisite: There is only 1 transaction stored in Spend N Split. The payer is "Roy".
+
+`listTransaction n=Roy`
+
+Expected Output in the Transaction List Panel: The transaction is displayed.
+
+Expected Output in the Command Output Panel: List transaction message is shown and states "1 transaction(s) listed".
+
+`listTransaction Lim`
+
+Expected Output in the Transaction List Panel: Nothing is displayed.
```

Comment:
The listTransaction command looks for a transaction with description containing the word Lim. It is not stated in the prerequisite what the description of the existing transaction is.

Reply:
good catch. thanks